example/router.lua

```diff
@@ -14,11 +14,13 @@ replicasets = {'cbf06940-0790-498b-948d-042b62cf3d29',
 
 -- Call a configuration provider
 cfg = dofile('localcfg.lua')
+-- vshard.router.cfg doesn't support box.cfg options.
+cfg.replication_connect_quorum = nil
```

Comment:
I suggest to use `vshard.cfg.extract_vshard()` for this. I will make sure the config only has vshard options.